pkgs/development/python-modules/django-rich/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  django,
+  rich,
+  pytestCheckHook,
+  pytest-django,
+}:
+
+buildPythonPackage rec {
+  pname = "django-rich";
+  version = "1.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "adamchainz";
+    repo = "django-rich";
+    rev = "refs/tags/${version}";
+    hash = "sha256-2Pf40+SxjaG9h4clrscnGOmyXoJLJGWM02iw1SdaQpE=";
+  };
+
+  postPatch = ''
+    # Disable coverage testing
+    echo "" > tests/testapp/__init__.py
+  '';
```

Comment:
Use pytest-cov-stub instead.

Reply:
It's using `coverage`, not `pytest-cov`, which is why I used this patch instead.